testing/events_test.go

```diff
@@ -0,0 +1,223 @@
+package ibctesting_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	abci "github.com/cometbft/cometbft/abci/types"
+
+	"github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+)
+
+func TestParsePacketsFromEvents(t *testing.T) {
+	tests := []struct {
```

Comment:
nit: this is the naming we use in the majority of other tests.

```suggestion
	testCases := []struct {
```